Makefile

```diff
@@ -52,7 +56,7 @@ gomod-check:
 
 ## Runs eslint and golangci-lint
 .PHONY: check-style
-check-style: gomod-check golangci-lint webapp/node_modules
+check-style: apply gomod-check golangci-lint webapp/node_modules
```

Comment:
Unrelated but since we are here can we invert and do the `gomod-check` as last? I've figured out it's super annoying when you are working on some uncommitted changes that import new things and you can't run `make check-style` cause it will always fail.